Source/Library/GameModel.cpp

```diff
@@ -2,16 +2,41 @@
 
 using namespace TicTacToe;
 
-GameModel::GameModel() {}
+GameModel::GameModel() : 
+  m_board{},
+  m_state{State::INVALID},
```

Comment:
By initializing this to ```State::CONTINUE``` instead of ```State::INVALID``` you can remove line 34 and in turn convert line 35 into an ```else if```.